javascript/node/selenium-webdriver/lib/virtual_authenticator.js

```diff
@@ -22,27 +22,26 @@
  * @enum {string}
  */
 const Protocol = {
-  'CTAP2': 'ctap2',
```

Comment:
Some tests use this, like `virtualAuthenticator_test.js`, so this is probably why CI is not passing. Can you please double check?

Same for `Transport` below.

Reply:
@diemol yes sure, let me check that

Reply:
@diemol i checked tests locally and everything is fine 
```../../../bazel-bin/javascript/node/selenium-webdriver/lib/atoms/get-attribute.js
../../../bazel-bin/javascript/node/selenium-webdriver/lib/atoms/is-displayed.js
../../../bazel-bin/javascript/node/selenium-webdriver/lib/atoms/find-elements.js
[INFO] Using Selenium remote end: http://localhost:4444/wd/hub/
[INFO] Running tests against [chrome]


  [chrome]
    VirtualAuthenticator Test Suit 2
      ✓ should test create authenticator (352ms)
      ✓ should test remove authenticator
      ✓ should test add non-resident credential (220ms)
      ✓ should test add non-resident credential when authenticator uses U2F protocol (220ms)
      ✓ should test add resident credential (231ms)
      ✓ should test add resident credential not supported when authenticator uses U2F protocol
      ✓ should test get credentials (349ms)
      ✓ should test remove credential by rawID (352ms)
      ✓ should test remove credential by base64url Id (348ms)
      ✓ should test remove all credentials (467ms)
      ✓ should test set user verified (465ms)


  11 passing (21s)```